### DISCUSSION TITLE
Habbo hotel like education as penalty

### DISCUSSION THREAD

**Discussion by aklidata** [PullDating/Org#83] [Ideas & Features]
instead of banning creepy users we could force them to read up an ethics manifest/training and take a quiz before they're allowed back on the app ( i got this idea from habbo hotel)

**Comment by WilliamDormer** [comment]
In principle, I like the idea of helping people change for the better. But I think that the effectiveness of this might be rather low, especially since a lot of adults will resent you making them read an HR document to correct their behaviour.
